wolfssl/wolfcrypt/settings.h

```diff
@@ -1808,6 +1808,37 @@ extern void uITRON4_free(void *p) ;
     #define SSL_CTRL_SET_TLSEXT_HOSTNAME
 #endif
 
+#if !defined(NO_DH) && !defined(HAVE_FFDHE)
+    #if defined(HAVE_FFDHE_2048) || defined(HAVE_FFDHE_3072) || \
+            defined(HAVE_FFDHE_4096) || defined(HAVE_FFDHE_6144) || \
+            defined(HAVE_FFDHE_8192)
+        #define HAVE_FFDHE
+    #endif
+#endif
+#ifdef FP_MAX_BITS
```

Comment:
Should we detect whether this was user set explicitly versus us setting as a default.  Say a user only has so much memory and has set FP_MAX_BITS to handle that size.  Should we be increasing the size w/o a warning or detection?

Reply:
Not sure what problem I was trying to solve here.
Changed to be a compiler error.
I don't want the user to compile in parameters that will cause random overruns and random failures.

Reply:
Found the configuration that doesn't work!
It is a customer configuration - WOLFSSL_X86_64_BUILD set FP_MAX_BITS.
Fixing that now.

Reply:
Fixed with latest push